src/olympia/activity/views.py

```diff
@@ -182,10 +182,17 @@ def download_attachment(request, log_id):
     Download attachment for a given activity log.
     """
     log = get_object_or_404(ActivityLog, pk=log_id)
+    addon = get_object_or_404(AddonLog, activity_log=log).addon
     attachmentlog = log.attachmentlog
 
-    is_reviewer = acl.action_allowed_for(request.user, amo.permissions.ADDONS_REVIEW)
-    if not is_reviewer:
+    is_reviewer = acl.is_user_any_kind_of_reviewer(request.user, allow_viewers=True)
```

Comment:
@wagnerand just confirming you're okay with every flavour of reviewer - and viewers - being able to download the attachments.

Reply:
Content reviewers likely won't really need to, but I am ok with every flavor of reviewer and viewer.